### PR TITLE
Test License Year

### PR DESCRIPTION
**Pull request by nolakat**
# Fixes # 

### Checklist
- [1] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [1] I have read the [Contribution Guide] and my PR follows them.
- [1] I updated my branch with the master branch.
- [1] I have added tests that prove my fix is effective or that my feature works
- [] I have added necessary documentation about the functionality in the appropriate .md file
- [1] I have added in line documentation to the code I modified

### Short description of what this PR does:
- 
- 

If you have questions, please send an email to [Sendgrid](mailto:dx@sendgrid.com), or file a Github Issue in this repository.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/smtpapi-csharp?pullRequest=63) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/smtpapi-csharp?pullRequest=63) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/smtpapi-csharp?pullRequest=63) it.</sub>

File: Smtpapi/Smtpapi/Utils.cs [hunk 1]
```diff
@@ -65,5 +66,21 @@ public static int DateTimeToUnixTimestamp(DateTime dateTime)
...
             return (int)span.TotalSeconds;
         }
+
+       
+        /// <summary>
+        /// Checks if string is current year
+        /// </summary>
+        /// <param name="license-date"></param>
+        /// <returns></returns>
+        public static string TestLicenseFileYear(string src)
```
Comment (mbernier) [inline Smtpapi/Smtpapi/Utils.cs]: This might be my ignorance on csharp, but I wanted to ask - is it necessary to have this in two places?

**Comment by thinkingserious** [comment]
@nolakat

We have not been able to merge your Pull Request, but because you are awesome - we wanted to make sure you could still get a SendGrid Hacktoberfest shirt.  

Please go **fill [out our swag form](https://dx.sendgrid.com/swag) before Nov 5th** and we will send the shirt! (We know that you might have tried this before and it didn’t work, sorry about that!)

### **You have till Nov 5th to fill out this form** in order to get the Hacktoberfest shirt!

Thank you for contributing during Hacktoberfest! We hope to see you in the repos soon! Just so you know, we always give away a SendGrid shirt for your first ever non-Hacktoberfest PR that gets merged.

File: Smtpapi/HeaderTests/TestUtils.cs [hunk 2]
```diff
@@ -57,5 +58,17 @@ public void TestEncodeNonAsciiCharacters()
 
             Assert.AreEqual(expected, result);
         }
+
+        [Test]
+        public void TestLicenseFileYear()
+        {
+            string test = "Copyright (c) 2014-2016 SendGrid, Inc.";
```
Comment (mbernier) [inline Smtpapi/HeaderTests/TestUtils.cs]: Can you please modify this to pull this string from the License file in the repository?

**Comment by childish-sambino** [comment]
Duplicates https://github.com/sendgrid/smtpapi-csharp/pull/59

**Pull request closed without merging**